client/src/routes/Publish/Files/Ipfs/Status.module.scss

```diff
@@ -7,13 +7,13 @@
 
     &:before {
         content: '';
-        width: .5rem;
-        height: .5rem;
+        width: 0.5rem;
+        height: 0.5rem;
         display: inline-block;
         background: $yellow;
         border-radius: 50%;
         margin-right: $spacer / 6;
-        margin-bottom: .1rem;
+        margin-bottom: 0.1rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)